.changelog/1242.feature.md

```diff
@@ -0,0 +1 @@
+Support limiting scope to one network or layer
```

Comment:
This seems hard to maintain. Did any other approaches have issues?

Reply:
> This seems hard to maintain.

Well, I guess we can simplify it a great deal if we give up having the shortest possible URLs.... lemme prepare and alternate PR with that version.

Reply:
How about #1269?

Reply:
It's not a different approach tho

Some other approaches:
- Do we only need to support one of `pinning network | pinning layer | pinning network+layer` for PontusX? Does it matter? Are there issues supporting just one?
- Issues if we tried to unify "fixed" vs unfixed by moving enabled layers to .env.production?
- Can react router intercept and rewrite paths
- ..